k8s-direct/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "dataplane-k8s-direct"
```

Comment:
Why `k8s-direct`? I understand we want the dataplane to talk directly to K8s _instead of using the current gateway agent_, but once this is in place and the agent is gone, the name will remain, and I'm not sure the `-direct` part will make more sense. How about something like `k8s-interface`? `k8s-crds`?

Reply:
package name is still dataplane-k8s-direct, only dir is renamed

Reply:
Fixed.